src/main/java/subway/line/dto/LineSectionAppendRequest.java

```diff
@@ -0,0 +1,26 @@
+package subway.line.dto;
+
+public class LineSectionAppendRequest {
+  private Long upStationId;
+  private Long downStationId;
+  private int distance;
+
+  public LineSectionAppendRequest() {
+  }
+
+  public LineSectionAppendRequest(Long upStationId, Long downStationId, int distance) {
+    this.downStationId = downStationId;
+    this.upStationId = upStationId;
+    this.distance = distance;
+  }
+  public Long getUpStationId() {
+    return upStationId;
+  }
+  public Long getDownStationId() {
```

Comment:
여기도 코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
  }
  
  public Long getUpStationId() {
    return upStationId;
  }
  
  public Long getDownStationId() {
```